.github/pull_request_template.md

```diff
@@ -29,9 +29,9 @@ Pull Request
 
 **How did you test the PR changes?** 
 <!--- Proof of testing is required in order for the PR to be approved. -->
-<!--- To check a box, remove the space and insert an x in the box (eg. [x] CircleCi). --> 
+<!--- To check a box, remove the space and insert an x in the box (eg. [x] Buildkite). --> 
```

Comment:
I will fix this in the migrations, so you guys don't need to worry. I figured that if we're releasing this onto main right now, we should update this so that it's up to date.